modules/DoubleClick/resources/mw.DoubleClick.js

```diff
@@ -1680,6 +1680,7 @@
 		destroy:function(){
 			// remove any old bindings:
 			var _this = this;
+			clearTimeout(_this.skipTimeoutId);
 			if ( this.adTagUrl || this.currentAdSlotType === "postroll" ){
```

Comment:
@dan-ziv - please use something like:
`			if( this.skipTimeoutId !== null ){
				clearTimeout(this.skipTimeoutId);
				this.skipTimeoutId = null;
			}`